src/applications/yellow-ribbon/components/SearchResultsPage/index.js

```diff
@@ -60,7 +60,7 @@ export const SearchResultsPage = ({
 
         {/* Helpful Links */}
         <div className="vads-u-display--none medium-screen:vads-u-display--flex vads-u-flex-direction--column">
-          <h3 className="vads-u-margin-top--2">Helpful links</h3>
+          <h2 className="vads-u-margin-top--2">Helpful links</h2>
```

Comment:
This change from `h3` to `h2` should not be made until after helpful links have been relocated to after the search results 🙇

Reply:
Okay! Got it, thank you.